src/secret.js

```diff
@@ -11,47 +12,62 @@
 
 "use strict";
```

Comment:
Think you can remove this now, too (I see you did that elsewhere, as well).

Reply:
Ah yes, thanks!